test/python/com/actioniq/play/tasks/test_routes_gen.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import
+
+
+from com.actioniq.play.tasks import routes_gen
+
+def mock():
+    """a simple object you can add properties to.
+    replace with unittest.mock when this is moved to python3
+    """
+    return lambda *args, **kwargs: None
+
+def test_routes_gen():
+    options = mock()
+    options.bootstrap = True
+    options.scope = ''
+    routes_gen.RoutesGen.register_options(options)
```

Comment:
Are there more assertions to be made here?

Reply:
I'm not sure what to assert here, but this does at least exercise the module a little and shows it doesn't blow up. Basically that will help give me confidence that additions like `from __future__ import absolute_import` didn't break anything.

Reply:
Makes sense to me, has there been any thought to committing this upstream also?

Reply:
I'd like to make a PR upstream. I might have to do that separately because we've diverged a bit in this fork.